api/src/index.ts

```diff
@@ -90,6 +90,8 @@ export { createTraceState } from './trace/internal/utils';
 export { TracerProvider } from './trace/tracer_provider';
 export { Tracer } from './trace/tracer';
 export { TracerOptions } from './trace/tracer_options';
+export { wrapTracer, SugaredTracer } from './experimental/trace/SugaredTracer';
+export { SugaredSpanOptions } from './experimental/trace/SugaredOptions';
```

Comment:
These exported names should be moved to a new entry file as listed in the package.json.